packages/react/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@axa-fr/react-oidc",
   "version": "6.15.8",
   "private": false,
```

Comment:
Thank you @jafin, I have got more time today to check.
To publish to npm all files. You need to updated "files" 
`
  "files": [
    "dist",
    "bin",
    "src/oidc",
    "service_worker",  <--- add this
    "src/override",
    "README.md",
    "package.json",
    "package-lock.json"
  ],
`